charts/palisade-service/templates/deployment.yaml

```
@@ -62,20 +62,27 @@ spec:
           {{- toYaml .Values.resources | nindent 10 }}
         env:
         - name: "LOADER_PATH"
-          value: {{ include "palisade-service.deployment.path" . }}
-        {{- $envVars := pluck .Values.global.deployment .Values.global.env | first | default .Values.global.env._default }}
-        {{- range $item := $envVars }}
+          value: {{ include "filtered-resource-service.deployment.path" . }}
+        - name: "SPRING_CONFIG_ADDITIONAL_LOCATION"
+          value: {{ include "filtered-resource-service.config.path" . }}
```

`filtered-resource-service`, are you sure about that?
```suggestion
          value: {{ include "palisade-service.deployment.path" . }}
        - name: "SPRING_CONFIG_ADDITIONAL_LOCATION"
          value: {{ include "palisade-service.config.path" . }}
```